support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -201,6 +202,15 @@ function getParticipations(
       return;
     }
 
+    if (test.urlParticipationOnly) {
+      const participationsFromUrl = getParticipationsFromUrl();
+      if (participationsFromUrl && participationsFromUrl[testId]) {
+        participations[testId] = participationsFromUrl[testId];
```

Comment:
would this be overwritten on line 215? and is that intentional?

Reply:
good point, it should return here